AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -49,8 +49,8 @@ When you put ABC files in the Assets folder under your Unity Project, Unity auto
 |:---|:---| 
```

Comment:
The table below has a lot of badly formed English sentences. Can you re-work them a bit and push a new version please?